src/router_sup.erl

```diff
@@ -137,7 +142,8 @@ init([]) ->
             ?SUP(router_console_sup, []),
             ?SUP(router_decoder_sup, []),
             ?WORKER(router_device_devaddr, [#{}]),
-            ?WORKER(router_xor_filter_worker, [#{}])
+            ?WORKER(router_xor_filter_worker, [#{}]),
+            ?WORKER(router_ics_worker, [ICSOpts])
```

Comment:
Should this be started before the `console_sup` so we don't miss any messages?

Reply:
Hmm, well it needs the console worker to make the refetch request when it starts up. Nevermind.